rtc/beta-access.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Beta access
+title_nav: Beta access
+description: Beta specific access and setup
+keywords: rtc
+---
+
+{% assign beta_feature = "The RTC plugin" %}
```

Comment:
I know this is a draft, but this whole page feels very much like a marketing page. That's not suited for technical documentation. Maybe we should have this on the website and link to it from the documentation instead?

If not, then `you`, etc... isn't something that should be in technical documentation so we'll need to look at rewording it to remove all that.

Reply:
I have hopefully addressed most of this in https://github.com/tinymce/tinymce-docs/pull/1801/commits/b1c8bf02e467e8b197c433163c57f9ddc977262b and brought the content more in-line with [our style guide](https://www.tiny.cloud/docs/configure/contributing-docs/#styleguide).

Reply:
LGTM, thanks! :+1: